Lib/ast.py

```diff
@@ -149,6 +155,8 @@ def _format(node, level=0):
                     keywords = True
                     continue
                 value, simple = _format(value, level)
+                if not show_empty and value in empty_values:
```

Comment:
Why not check this before the `_format` call? That way we don't have to look at the string value.